docs/reference/modules/network.asciidoc

```diff
@@ -35,11 +34,23 @@ in the following table:
 
 |`_local:ipv6_` |Will be resolved to loopback IPv6 addresses
 
-|`_non_loopback_` |Addresses of the first non loopback interface
+|`_site_` |Will be resolved to site-local addresses ("private network")
```

Comment:
Maybe put an example here instead of `("private network")`? Like `(e.g. 192.168.0.1, 172.28.128.3, 10.123.1.28)`?